examples/kafka-broker/docker-compose.yaml

```diff
@@ -6,8 +6,6 @@ services:
       context: .
       dockerfile: Dockerfile-proxy
     container_name: proxy
-    depends_on:
-    - kafka
```

Comment:
i think we do want this - and we want to make it wait for kafka to be healthy

Reply:
I think the Envoy can be started first and the cluster's health check interval can help. Correct me if I'm wrong

Reply:
i think best to make envoy start last and only after the others have come up

that way you can be pretty confident that when you call it via the proxy it will respond